src/components/H5PElement/H5PElement.jsx

```diff
@@ -103,6 +105,14 @@ class H5PElement extends Component {
 H5PElement.propTypes = {
   h5pUrl: PropTypes.string,
   onSelect: PropTypes.func.isRequired,
+  locale: PropTypes.string.isRequired,
 };
 
-export default injectT(H5PElement);
+const mapStateToProps = state => {
```

Comment:
Bruk heller `LocaleContext`. Jeg lagde for en stund tilbake `LocaleContext`i App.jsx for å sakte gå vekk fra redux. Siden du er i en klassekomponent så brukes den ved å wrappe `<LocaleContext.Consumer> </LocaleContext.Consumer>` rundt komponenten også får du `locale` som prop inn i den. Ser at du bruker locale som prop utenfor komponenten, da ville jeg anbefalt å gjøre komponenten om til FunctionComponent for da kan du bare bruke den hvor som helst slik:

```
const locale = useContext(LocaleContext);
```

Reply:
Skal teste!

Reply:
Dvs eg lagde det ikkje om til komponent, men funka fint med consumer.